src/app/components/data-access-panel/data-access-panel.component.scss

```diff
@@ -10,6 +10,7 @@
     .get-data-text {
         color: #1a7f37;
         font-weight: 700;
+        font-size: 17px !important;
```

Comment:
I remember Oleksiy was reacting bad on `!important`. Is it strictly necessary?

Reply:
Yes, fixed